impl/src/main/resources/tck-audit-cdi.xml

```diff
@@ -5157,6 +5157,13 @@
             </text>
         </assertion>
 
+        <assertion id="l">
+            <text>The Priority annotation can be used to enable and order interceptors associated with components that use interceptor bindings.
```

Comment:
I don't mind this, just FTR - there is a `tck-audit-int` file which will probably already have a section related to this.

Reply:
`tck-audit-int` has an `id-`less assertion in `priority_annotation_in_ordering_int` section, with a note that "Tested within the CDI TCK, section 9.4 Interceptor enablement and ordering", so I'm not entirely sure: should I quote this assertion from `tck-audit-int` instead? Adding a new assertion to the "Interceptor enablement and ordering" section of `tck-audit-cdi` seemed a bit better to me, but I'm not really sure.

Reply:
I see In that case, the current solution seems good to me